rfcs/any.js-globals.md

```diff
@@ -0,0 +1,14 @@
+# RFC ..: Remove default and negated global support in *.any.js
+
+## Summary
+
+https://web-platform-tests.org/writing-tests/testharness.html#multi-global-tests has a confusing syntax for specifying globals. E.g., `global=window` is equivalent to `global=window,dedicatedworker` and `global=jsshell` is equivalent to `global=window,dedicatedworker,jsshell`.
+
+## Details
+
+1. Update all `*.any.js` tests that use `// META: global=` to list all the relevant globals explicitly when possible. (Some necessarily rely on `!default` or `!` (negation).
+2. Atomically:
+   1. Remove support for `default` and `!`.
+   1. Update all remaining `*.any.js` tests that use `!` to list all the relevant globals explicitly.
+   
+https://github.com/web-platform-tests/wpt/issues/23111 tracks this work.
```

Comment:
Are there any risks that should be detailed here? I think the main one is just downstream implementers as usual. I'm happy to own checking the impact this will have on chromium.

Reply:
I guess the risk is getting the changes wrong. I don't see any downstream risks personally, but I'm not aware of all downstream usage either.

Reply:
The downstream risk would be implementors who are not using the upstream infrastructure who could have reimplemented part of the logic here. I'm not specifically aware of anyone doing that (gecko doesn't) but it's at least possible.